src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -377,6 +377,12 @@ export enum BinMethodOption {
     CUSTOM = 'CUSTOM',
 }
 
+type CustomDataType = 'CATEGORICAL' | 'NUMERICAL';
```

Comment:
https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-compile-time
this should allow you to avoid this duplication